shared/actions/config/index.tsx

```diff
@@ -327,6 +336,16 @@ function* maybeDoneWithLogoutHandshake(state) {
 let routeToInitialScreenOnce = false
 
 const routeToInitialScreen2 = (state: Container.TypedState) => {
+  if (
+    state.config.loggedIn &&
+    state.config.startupLink &&
+    state.config.startupLink.endsWith('/phone-app') &&
+    !state.settings.phoneNumbers.phones
+  ) {
+    // pending loadMySettings finishing
+    return
```

Comment:
lets not do this. just go there and show a spinner

Reply:
im not sure if i understand this, could you please clarify?